poetry.lock

```diff
@@ -1,10 +1,9 @@
-# This file is automatically @generated by Poetry 1.4.2 and should not be changed by hand.
+# This file is automatically @generated by Poetry 1.5.1 and should not be changed by hand.
 
 [[package]]
 name = "amqp"
 version = "5.1.1"
 description = "Low-level AMQP client for Python (fork of amqplib)."
-category = "main"
```

Comment:
Looks like the categories got removed from the lockfile when I ran `poetry update`. Is this a problem?

Reply:
https://github.com/python-poetry/poetry/issues/5702#issuecomment-1139512812 It seems like `categories` are going to be dropped in a future release anyways so shouldnt matter much? Do double check me on this.